tests/unit/table/table_core.js

```diff
@@ -15,7 +15,7 @@ QUnit.module( "Basic Table", {
 				QUnit.start();
 			} );
 
-			$.mobile.changePage( hash );
+			$( ".ui-pagecontainer" ).pagecontainer( "change", hash );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?